tests/integration/targets/ios_lldp_interfaces/tests/cli/overridden.yaml

```diff
@@ -7,6 +7,8 @@
 - ansible.builtin.include_tasks: _remove_config.yaml
 
 - ansible.builtin.include_tasks: _populate_config.yaml
+  vars:
+    isMerged: true
```

Comment:
why are we introducing this parameter? is it specific to some test failure?

Reply:
yes